cardano-cli/cardano-cli.cabal

```diff
@@ -203,12 +203,7 @@ library
                       , cardano-crypto-wrapper ^>= 1.5.1
                       , cardano-data >= 1.1
                       , cardano-git-rev
-                      , cardano-ledger-alonzo >= 1.5.0.0
-                      , cardano-ledger-binary >= 1.2
                       , cardano-ledger-byron >= 1.0.0.2
```

Comment:
Would it be possible to remove this dependency too?

Reply:
I am not sure, I think I found two data-types with the same name when trying. But I would prefer to attempt it in a separate PR